### DISCUSSION TITLE
❌ Generated Postman Collections?

### DISCUSSION THREAD

**Discussion by samburgers** [RobinTail/express-zod-api#310] [Ideas]
Wonder if it would be possible to generate postman collections for the endpoints somehow 🙏 

https://github.com/postmanlabs/postman-collection

**Comment by samburgers** [comment]
Or actually maybe this 

https://github.com/postmanlabs/openapi-to-postman

**Reply by samburgers** [reply]
^ can confirm this package works well (aside from not having types)

**Comment by RobinTail** [comment]
Hello @samburgers .
I don't plan such integration in the near future. 
Try to use the package you found to convert the specification that the library generates.
